src/common/common.h

```diff
@@ -138,11 +138,16 @@ typedef enum {
 OQS_API int OQS_CPU_has_extension(OQS_CPU_EXT ext);
 
 /**
- * This currently only sets the values in the OQS_CPU_EXTENSIONS,
- * and so has effect only when OQS_DIST_BUILD is set.
+ * This currently sets the values in the OQS_CPU_EXTENSIONS
+ * and prefetches the OpenSSL obejcts if necessary.
```

Comment:
nit: typo. Can deal with when doing next doc review. Would also recommend phrasing more strongly along the lines of "Needs to be invoked to ensure proper operation of library."